tests/unit/amo/components/TestAddonReviewList.js

```diff
@@ -530,15 +531,23 @@ describe(__filename, () => {
       dispatchAddon();
       dispatchAddonReviews();
       const root = render();
-      expect(root.find(Paginate))
-        .toHaveProp('pathname', root.instance().url());
+
+      const footer = root.find('.AddonReviewList-reviews').prop('footer');
+      const paginator = shallow(footer);
+
+      expect(paginator.instance()).toBeInstanceOf(Paginate);
+      expect(paginator).toHaveProp('pathname', root.instance().url());
     });
 
     it('configures a paginator with the right Link', () => {
       dispatchAddon();
       dispatchAddonReviews();
+
       const root = render();
-      expect(root.find(Paginate)).toHaveProp('LinkComponent', Link);
+      const footer = root.find('.AddonReviewList-reviews').prop('footer');
+      const paginator = shallow(footer);
```

Comment:
Is it worth creating a `paginator` variable when you're only using it once below?

Reply:
I've done that for readability purpose, because shallowing a footer does not really mean it is a `paginator` under the hood.

Reply:
Added a comment instead of a variable.